pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java

```diff
@@ -72,6 +84,13 @@ public static void main(String[] args) throws InterruptedException {
 
         ZooKeeper localZk = initZk(arguments.zookeeper, arguments.zkSessionTimeoutMillis);
 
+        if (arguments.bkMetadataServiceUri != null) {
+            BookKeeper bookKeeper = new BookKeeper(new ClientConfiguration().setMetadataServiceUri(arguments.bkMetadataServiceUri));
```

Comment:
Can we use the org.apache.bookkeeper.client.api.Bookkeeper interface? 
You are using the old, soon deprecated, API

Reply:
IMO, it's better to be consistent with the project. The Pulsar project uses the old API currently which can be verified by running

```bash
find . -name "*.java" | xargs grep -n "import .*\.BookKeeper"
```
in the project directory.

Reply:
Makes sense.
We should create a ticket to movie to the new API.
I will take care of it.